benchmarks/dynamo/microbenchmarks/tensor_layout_mini_benchmark.py

```diff
@@ -54,8 +54,8 @@ def test_fn():
         test_out[0][0][0][:32],
     )
 
-    baseline_ms = do_bench(baseline_fn, rep=40)
-    test_ms = do_bench(test_fn, rep=40)
+    baseline_ms = benchmarker.benchmark(baseline_fn, (), {}, rep=40)
```

Comment:
Would be great if we could migrate callsites as needed instead of changing the api - necessitating args and kwargs in every call is an ergonomic downgrade

Reply:
Tbh I think this is already either not working or I'm missing something super obvious, because torch._inductor.runtime.runtime_utils.do_bench does already necessitate args and kwargs

Reply:
readded do_bench to runtime_utils, just redirects to benchmarker